frontend/common/code-help/offline_client/offline-client-curl.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (envId) => `
+curl 'https://edge.api.flagsmith.com/api/v1/flags/'\\
+     -H 'x-environment-key: ${envId}'>>flagsmith.json
```

Comment:
nit: I'd argue this should be:

```suggestion
     -H 'x-environment-key: ${envId}' > flagsmith.json
```

I get that it's probably a very slim use case, but it's odd that it appends rather than replaces the existing data. Also the lack of white space annoyed me 😄

Reply:
I've done this.